tests/test_modeling_common.py

```diff
@@ -1539,6 +1552,44 @@ def check_same_values(layer_1, layer_2):
             # self.assertTrue(model.transformer.wte.weight.shape, model.lm_head.weight.shape)
             # self.assertTrue(check_same_values(model.transformer.wte, model.lm_head))
 
+    def test_tied_model_weights_key_ignore(self):
```

Comment:
I would suggest to make the name more generic (i.e. not using `tied_model` in the name)

The check in this test is not really specific to tied weights, although they are one of the main source of missing keys.

`test_model_weights_key_ignore` + a comment to explain its objective would be fine.

Reply:
In my mind it is only about tied weights (because it's the only mechanism I know that can create the weight sharing).

Happy to update the name